benches/mantis/jobs/mantis.ncl

```diff
@@ -5,69 +5,83 @@ let lib = import "../lib.ncl" in
 
 # Temporary dummy value
 let baseTags = [""] in
+let Params =
+  {
+    count
+      | number.Nat
+      | default
+      = 5,
+    role | [| `passive, `miner, `backup |],
+    datacenters | Dyn,
+    job | Dyn,
+    namespace | String,
+    logLevel | String,
+    mantisRev | String,
+    fqdn | String,
+    loggers | { _ : String },
+    network
+      | lib.contracts.OneOf [ "testnet-internal-nomad", "etc" ]
```

Comment:
For example (continuing with the spacing), why does this array get spacing around braces, but `baseTags` above doesn't?

Reply:
(See above comment about single-element lists)